src/providers/test-data-by-category/test-data-by-category.ts

```diff
@@ -18,6 +19,7 @@ export class TestDataByCategoryProvider {
       case TestCategory.C1: return getTestDataC1;
       case TestCategory.CE: return getTestDataCE;
       case TestCategory.C1E: return getTestDataC1E;
+      case TestCategory.D: return getTestDataCatD;
```

Comment:
Might be worth adding the other D categories here, just to avoid errors in the future